perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -44,16 +44,27 @@ sudo dpkg -i $HOME/release/packages/gcsfuse_${GCSFUSE_VERSION}_amd64.deb
 cd "./perfmetrics/scripts/"
 
 GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs --log-format \"text\" --stackdriver-export-interval=30s"
-LOG_FILE_FIO_TESTS=${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs.txt
-GCSFUSE_FIO_FLAGS="$GCSFUSE_FLAGS --log-file $LOG_FILE_FIO_TESTS"
+
+cd "./bigquery"
```

Comment:
instead of changing directory, can we specify bigquery/requirements.txt ?

Using less cd makes the script easier to understand.